cert-manager/cert-manager-webhook-role-binding/base/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kube-system
```

Comment:
i believe this namespace will be over-ruled by the one kfctl generates at /manifests/cert-manager/cert-manager-webhook-role-binding/kustomization.yaml

Reply:
I'll strip it off here.

Reply:
This makes it easier for using with Kustomize - so I've retained it.